### PR TITLE
mcux:cmake: the fsl_clock uses SDK_DelayAtLeastUs in fsl_common.c

### PR DESCRIPTION
**Pull request by MarkWangChinese**
Signed-off-by: Mark Wang <yichang.wang@nxp.com>

### CODE REVIEW

**Comment by MarkWangChinese** [comment]
The #9 need this PR to fix linking error.

File: mcux/CMakeLists.txt [hunk 1]
```diff
@@ -32,6 +32,10 @@ zephyr_compile_definitions(${MCUX_CPU})
 # practice, drilling down like this avoids the need for repetitive
 # build scripts for every mcux device.
 zephyr_sources(devices/${MCUX_DEVICE}/fsl_clock.c)
+# fsl_colck use the functions in fsl_common.
```
Comment (galak) [inline mcux/CMakeLists.txt]: nit: fix typo in comment (fsl_colck -> fsl_clock)

Comment (MarkWangChinese) [reply mcux/CMakeLists.txt]: fixed

**Comment by MaureenHelm** [comment]
#38 added support for building `mcux/drivers/lpc/fsl_common.c`, so this PR is no longer needed.

**Pull request closed without merging**